### PR TITLE
Room owners and moderators received via method call then stream

### PR DESCRIPTION
**Pull request by marceloschmidt**

### CODE REVIEW

File: packages/rocketchat-ui/views/app/room.coffee [hunk 1]
```diff
@@ -493,7 +493,40 @@ Template.room.onCreated ->
...
 
-	@subscribe 'roomModerators', @data._id
+	Meteor.call 'getRoomModeratorsAndOwners', @data._id, (error, results) ->
+		if error
+			return toastr.error error.reason
+
+		for record in results
+			RoomModeratorsAndOwners.insert record
+
+	RocketChat.callbacks.add('streamMessage', (msg) ->
```
Comment (rodrigok) [inline packages/rocketchat-ui/views/app/room.coffee]: @marceloschmidt These callbacks will be added (duplicated) for each new opened room?

File: packages/rocketchat-ui/views/app/room.coffee [hunk 2]
```diff
@@ -493,7 +493,40 @@ Template.room.onCreated ->
 	@autorun =>
 		@subscribe 'fullUserData', Session.get('showUserInfo'), 1
 
-	@subscribe 'roomModerators', @data._id
+	Meteor.call 'getRoomModeratorsAndOwners', @data._id, (error, results) ->
+		if error
+			return toastr.error error.reason
+
+		for record in results
+			RoomModeratorsAndOwners.insert record
```
Comment (rodrigok) [inline packages/rocketchat-ui/views/app/room.coffee]: This code will throw an exception if the room expires and open again, will insert duplicated IDs, you could user upsert here.

File: packages/rocketchat-ui/views/app/room.coffee (hunk 1 above)
Comment (marceloschmidt) [inline packages/rocketchat-ui/views/app/room.coffee]: No, because I'm giving the callback an id: "addOrRemoveModerator" and "addOrRemoveOwner"

Comment (rodrigok) [inline packages/rocketchat-ui/views/app/room.coffee]: Sure, but I think you can move that code to some place that call only one time and prevent this processing, right? It's not necessary to execute this callback more than one time, or am I wrong?

**Comment by rodrigok** [comment]
LGTM

**Pull request merged**